### ISSUE TITLE
Backups: Index restoration issues

### ISSUE THREAD

**Issue by dfdan** [inveniosoftware/invenio-app-rdm#1816]
I've attempted to restore a backup in a DR scenario, using only backed up:

* instance directory (for config, custom vocabs/templates etc)
* SQL dump 
* attached files 

I believe this is what is deemed to be required for DR purposes? I was able to rebuild my venv, re-import the data/files fine. However, when restoring the instance, there doesn't seem to be a CLI command (ie available to an administrator) way of rebuilding certain key indices making a complete DR restore challenging. 

* `invenio communities rebuild-index` only rebuilds the main `communities-communities` index. Members and archivedinvitations are *not* restored. This means there are no members showing in any communities (at least via the UI).

* There is no CLI command to rebuild the requests indices. This again causes important things not to be visible in the UI. There needs to be a way to rebuild these indcies. 

Tested on 9.0+9.1. Cc: @kpsherva, @fenekku who I've discussed this with already.

**Comment by fenekku** [comment]
I haven't tried the following in a recovery scenario, but what about: `pipenv run invenio index reindex`? Does it re-index *everything* and address our problems?

**Comment by dfdan** [comment]
I don't think so - `invenio index reindex` requires you specify a pid_type (with -t) - so I don't think its applicable to community members / requests? 

I will do some more testing when I get a chance though. I'd forgotten about this command since `rdm-records rebuild-index` came along. This might help reindexing stuff that's stored in vocabularies though (licences, languages etc)?
